lib/CMakeLists.txt

```diff
@@ -51,6 +51,12 @@ set(SOURCES
 add_llvm_loadable_module(Svf ${SOURCES})
 add_llvm_Library(LLVMSvf ${SOURCES})
 
+# It seems that the Svf library needs to explictly require the CUDD alloc/init/etc so LLVMCudd
```

Comment:
Sure, this ought to be straightforward I think.  I just moved computers but let me create a VM to test in and put in a architecture check and I think this can be resolved... (I'll try to update tonight).